Pod/Classes/Logging/Log.swift

```diff
@@ -74,37 +113,105 @@ open class Log {
     }()
 
     /// Generic log method
-    fileprivate static func log<T>(_ object: @autoclosure () -> T, level: Log.Level, _ fileName: String, _ functionName: String, _ line: Int) {
-        if logLevel.rawValue <= level.rawValue {
-            let date = Log.dateformatter.string(from: Date())
-            let components: [String] = fileName.components(separatedBy: "/")
-            let objectName = components.last ?? "Unknown Object"
-            let levelString = Log.useEmoji ? level.emoji : "|" + level.name.uppercased() + "|"
-            let logString = "\(levelString)\(date) \(objectName) \(functionName) line \(line):\n\(object())"
-            print(logString + "\n")
-            handler?(level, logString)
+    internal func log<T>(_ object: @autoclosure () -> T, level: Log.Level, _ fileName: String, _ functionName: String, _ line: Int) {
+        guard logLevel.rawValue <= level.rawValue else {return}
+
+        let date = Log.dateformatter.string(from: Date())
+        let components: [String] = fileName.components(separatedBy: "/")
+        let objectName = components.last ?? "Unknown Object"
+        let levelString = useEmoji ? level.emoji : "|" + level.name.uppercased() + "|"
+        let logString = "\(levelString)\(date) \(objectName) \(functionName) line \(line):\n\(object())"
+
+        if #available(iOS 14.0, *) {
+            let objectString = "\(object())"
+            let logMessage = "\(objectName) \(functionName) line \(line)"
+            osLog(logMessage, objectString: objectString, level: level)
+        }
+        else {
+            let nameString = name.count > 0 ? "[\(name)]]" : ""
+            print(nameString + logString + "\n")
         }
+        self.handler?(level, logString)
+        Log.globalHandler?(self, level, logString)
     }
 
-    // MARK: Log Methods
+    @available(iOS 14.0, *)
+    internal func osLog(_ logMessage: String, objectString: String, level: Level) {
+        let logger = Logger(subsystem: subsystem, category: name)
```

Comment:
CI seems to be having difficulties with this line, even though it's flagged as available only in 14.0